webapp_frontend/src/components/basicElements/Loading.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import Spinner from "react-bootstrap/Spinner"
+
+
+export function FFLoading(): JSX.Element {
+    return (
+        <Spinner animation="border" role="status">
+            <span className="sr-only">Loading...</span>
+        </Spinner>
+    );
+}
```

Comment:
Pretty sure this is the wrong place. also this isn't styled properly. (location)